model.py

```diff
@@ -37,8 +37,8 @@ def train(validate=True, epochs=20):
     X = []
     y = []
 
-    X, y = preprocess.populate(X, y, "data/labeled/yes")
-    X, y = preprocess.populate(X, y, "data/labeled/no", end=True)
+    X, y = preprocess.populate(X, y, "data/labeled/yes", use_nir=True)
+    X, y = preprocess.populate(X, y, "data/labeled/no", use_nir=True , end=True)
```

Comment:
```suggestion
    X, y = preprocess.populate(X, y, "data/labeled/yes", use_nir=use_nir)
    X, y = preprocess.populate(X, y, "data/labeled/no", use_nir=use_nir , end=True)
```
Pass the original parameter value from the function so you don't hard code it .